notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -0,0 +1,115 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1: Bayesian refresher and Introduction to ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_What does an end to end Bayesian Workflow look like?_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning Objectives\n",
+    "* Refresh our understanding of Bayes Theorom\n",
+    "* Fit a small binomial model\n",
+    "* Show how a full statistical workflow, even outside of Bayesian Methods, requires more steps more than just model fitting\n",
+    "* Introduce ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayes Theorem\n",
+    "### The most common formulation\n",
+    "$$\n",
+    "\\Large\n",
+    "P(\\theta | y) = \\frac{ P(y) | \\theta)p(\\theta)}{p(y}\n",
+    "$$\n",
+    "\n",
+    "### Breaking it down\n",
+    "####  $P(\\theta | y)$ -> Posterior distribution\n",
+    "_\"What is the distribution of parameters given the observed data?\"_  \n",
+    "\n",
+    "After obtaining data, or making observations, what is our belief regarding the parameters of the underlying statistical model? \n",
+    "\n",
+    "* Estimating the posterior distribution is the goal of Bayesian analysis. \n",
+    "* The process of estimating the posterior distribution often refered to as **Inference**\n",
+    "* There are numerous inference ways to perform inference, each with their own pros and cons\n",
+    "    * In this tutorial we will only be using Markov Chain Monte Carlo (MCMC)\n",
+    "\n",
+    "####  $P(y | \\theta )$ -> Likelihood\n",
+    "_\"What is the probability of the observed data given a model parameter value\"_  \n",
+    "\n",
+    "Likelihood functions tell us how \"likely\" the observed data is, for all the possible parameter values. It is important to note that likelihood is not a probability distribution, it is a function that gives us probability values. Likelihood can roughly be thought of as loss functions from \"Machine Learning\"\n",
+    "\n",
+    "####  $P(\\theta)$ -> Prior\n",
+    "_\"What is the probability of distribution parameter given no observations\"_  \n",
+    "\n",
+    "Before we've observed any data what is the probability distributions of possible parameters?\n",
+    "\n",
+    "####  $P(y)$ -> Marginal Probability of Evidence\n",
+    "\n",
+    "What is the probability distribution of data? In most cases this term is difficult or impossible to calculate, so much so that most inference techniques cleverly get around their calculations. MCMC is one of those techniques\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Alternative formulations\n",
+    "\n",
+    "### Likelihood notation\n",
+    "I particularly like this formulation because clearly demarcates difference between Likelihood and probability terms  \n",
```

Comment:
I find that most people find this distinction confusing. assigning all terms the same status makes Bayesian analysis conceptually easier. I do not remember the details, but I think Richard Mcelreath has a talk about "removing frequentists language" from Bayesian explanations. Also Watanabe provides a brief intro to Bayesian statistics without making the distinction, he just say "In Bayesian statistics, we set a statistical model p(x|w) and a prior φ(w),
where p(x|w) is a conditional probability density of x for a given parameter
w and φ(w) is a probability density of w. Both p(x|w) and φ(w) are prepared
by a statistician who does not know a true distribution. Hence they may be
quite different from or inappropriate for the true distribution."

Reply:
Problem I have when I went to a new book or paper I suddenly saw this new symbol and was immediately lost. In this tutorial I would like to note every formulation so if someone picks up this tutorial they can go to any book and and a common reference.

Even in the text above the use of a new symbol φ makes it persistently confusing to me if φ is the same as p

When everyone agrees life will be great, but when multiple versions exist I think we should show them all. 

One easy change  I can make is removing the text that "i like this the best". What do you think?